ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -113,4 +114,51 @@ export const LivechatEnterprise = {
 
 		return LivechatTag.createOrUpdateTag(_id, tagData, tagDepartments);
 	},
+
+	savePriority(_id, priorityData) {
+		check(_id, Match.Maybe(String));
+
+		check(priorityData, {
+			name: String,
+			description: Match.Optional(String),
+			color: String,
+			dueTimeInMinutes: String,
+		});
+		LivechatRooms.updatePriorityDataByPriorityId(_id, priorityData);
+		Subscriptions.updatePriorityDataByPriorityId(_id, priorityData);
+		LivechatInquiry.updatePriorityDataByPriorityId(_id, priorityData);
+		return LivechatPriority.createOrUpdatePriority(_id, priorityData);
+	},
+
+	removePriority(_id) {
+		check(_id, String);
+
+		const priority = LivechatPriority.findOneById(_id, { fields: { _id: 1 } });
+
+		if (!priority) {
+			throw new Meteor.Error('priority-not-found', 'Priority not found', { method: 'livechat:removePriority' });
+		}
+		LivechatRooms.unsetPriorityByPriorityId(_id);
```

Comment:
My suggestion is to run the models update just in case the `LivechatPriority.removeById` succeed.